sklearn/tree/export.py

```diff
@@ -458,10 +460,10 @@ def recurse(tree, node_id, criterion, parent=None, depth=0):
             out_file.write('rankdir=LR ;\n')
 
         # Now recurse the tree and add node & edge attributes
-        if isinstance(decision_tree, _tree.Tree):
-            recurse(decision_tree, 0, criterion="impurity")
-        else:
+        if hasattr(decision_tree, 'tree_'):
```

Comment:
This is already checked by the check_is_fitted call above

Reply:
ok, so I'll remove it.